### DISCUSSION TITLE
Direct messages in chat aren't ordered by recent activity

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#18461] [Q&A]
On osu-web the direct messages are order by recent activity (newer DMs are at the top), while in lazer it's seemingly random.

Direct messages order on web:
![osu-web](https://pek.li/u385uq.png)
vs. order in lazer:
![lazer](https://pek.li/rm4gck.png)

This isn't really an issue when you only have a few DMs open like I do in the screenshots, but the more chats you open the harder it gets to keep track of which chats are actually relevant.

**Comment by Infiniti727** [comment]
2024 and this is still a thing :/, i know we have search feature now, but is still annoying

**Reply by peppy** [reply]
It this only when you are actively messaging people? As in your expect it to reorder in realtime?

**Reply by Infiniti727** [reply]
yeah, as i said, the search feature kinda fixes this, but since the web version does this it would be nice to have this be a thing in lazer, i dont like having to scroll a lot of chats just to find the chat of someone i want to talk to if i dont remember their username but we talked recently
